### PR TITLE
Configure BrowserID login/logout.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: marketpulse/base/templates/base.html [hunk 1]
```diff
@@ -15,9 +15,10 @@
 
     <link rel="stylesheet" type="text/css" href="{{ static('css/bootstrap.min.css') }}" media="screen" />
     <link rel="stylesheet" type="text/css" href="{{ static('css/sandstone.min.css') }}" media="screen" />
-    <link href="{{ static('gaia/input_areas.css') }}" rel="stylesheet">
-    <link href="{{ static('gaia/switches.css') }}" rel="stylesheet">
     <link rel="stylesheet" type="text/css" href="{{ static('css/app.css') }}" media="screen" />
+    <link rel="stylesheet" href="{{ static('gaia/input_areas.css') }}">
+    <link rel="stylesheet" href="{{ static('gaia/switches.css') }}" >
+    <link rel="stylesheet" href="{{ static('browserid/persona-buttons.css') }}">
```
Comment (comzeradd) [inline marketpulse/base/templates/base.html:20]: Why did you move gaia styles lower? `app.css` should always be the last style.

Comment (johngian) [inline marketpulse/base/templates/base.html:20]: Oops wrong copy/paste. Nice catch :+1:
